handel/src/aws/pricing-calls.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2018 Brigham Young University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import awsWrapper from './aws-wrapper';
+
+function convertMemoryStringToMemoryUnits(inputString: string) {
+    // Converts the given GiB into the amazon units
+    const gibiBitesOfMemory = inputString.split('GiB', 1);
```

Comment:
It's probably good practice to call .trim() after doing the split so that you just pass the number in and no whitespace. It probably doesn't affect functionality here, but it could be used in future modifications where that would be important.